python/translit/converters/translit_cyr_taras_to_lat.py

```diff
@@ -1,4 +1,6 @@
 import re
+from translit.diff_mode import is_new
```

Comment:
the same here... do we need to have this import here?

Reply:
Done.